ow_lander/src/ow_lander/actions.py

```diff
@@ -189,6 +197,86 @@ def plan_trajectory(self, _goal):
     return self._planner.deliver_sample()
 
 
+class ArmMoveCartesianServer(ArmActionMixin, ActionServerBase):
+
+  name          = 'ArmMoveCartesian'
+  action_type   = owl_msgs.msg.ArmMoveCartesianAction
+  goal_type     = owl_msgs.msg.ArmMoveCartesianGoal
+  feedback_type = owl_msgs.msg.ArmMoveCartesianFeedback
+  result_type   = owl_msgs.msg.ArmMoveCartesianResult
+
+  def publish_feedback_cb(self):
+    self._publish_feedback(pose=self._arm_tip_monitor.get_link_pose())
+
+  def execute_action(self, goal):
+    ARM_END_EFFECTOR = 'l_scoop_tip'
```

Comment:
The end effector is really the scoop and the scoop tip is just a point on the scoop.  It there a good reason to use one over the other?

Reply:
If `l_scoop` was used then the origin of the scoop would be the point that gets posed, which I think is set to its center-of-mass. I don't do a ton of work with robotic arms, but I feel like it makes the most sense to instead pose the tip or the furthest out point of the end-effector. That is certainly what you would want to use for ArmFindSurface, otherwise the surface you find would appear to be inside the scoop.